Probesys/bileto
# imp: Make roles easier to understand

## Related issue(s)



https://github.com/Probesys/bileto/issues/556

## How to test manually



1. Go to the roles page in the settings
1. Check there are three sections: Administrator roles, Operational roles, and User roles
1. Create an Operational role
1. Check there is a disabled (but checked) checkbox "Can be assigned to a ticket"
1. Create a User role
1. Check some of the permissions are not listed (see above for the full list)
2. Give an authorization to a user
3. Check that the label (Operational, User or Administrator) corresponds to the corresponding role

## Checklist
